doc/book/box/data_model.rst

```diff
@@ -261,15 +287,53 @@ Here are examples of numbers using regular notation, exponential notation,
 the ULL suffix and the ``tonumber64`` function:
 ``-55``, ``-2.7e+20``, ``100000000000000ULL``, ``tonumber64('18446744073709551615')``.
 
-.. _index-box_decimal:
+..  _index-box_decimal:
 
 **decimal**. The Tarantool decimal type is stored as a MsgPack ext (Extension).
 Values with the decimal type are not floating-point values although
 they may contain decimal points.
 They are exact with up to 38 digits of precision.
 Example: a value returned by a function in the :ref:`decimal <decimal>` module.
 
-.. _index-box_string:
+..  _index-box_datetime:
+
+**datetime**. Introduced in :tarantool-release:`2.10.0`.
+The Tarantool ``datetime`` type facilitates operations with date and time,
+accounting for leap years or the varying number of days in a month.
+It is stored as a MsgPack ext (Extension).
+This data type is based on the `c-dt <https://github.com/tarantool/c-dt>`_ module.
```

Comment:
Should be better sound like "Operations on this data type use code from  3rd party `c-dt <https://github.com/tarantool/c-dt>`_ library."